README.md

```diff
@@ -6,4 +6,37 @@ Samples](https://github.com/nvpro-samples/vk_video_samples) and [Conformance
 Test Suite](https://github.com/KhronosGroup/VK-GL-CTS), with one using
 [GStreamer](https://gstreamer.freedesktop.org/). It aims to be API compatible.
 
-Currently only supports H.264 streams. H.265 is in the pipeline.
+Currently supports H.264 anh H.265 streams.
+
+## Setup
+
+### Ubuntu/Debian/etc
+
+```sh
+apt install python3-pip ninja-build pkg-config
+pip3 install --user meson
+apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-bad
+```
+
+### Windows
+
```

Comment:
I would mention here that VS201x has to be installed. Perhaps linking with windows section of the gstreamer readme.